lib/cucumber/cli.js

```diff
@@ -51,7 +50,12 @@ function Cli(argv) {
 
   var self = {
     run: function run(callback) {
-      var configuration = getConfiguration();
+      var program = getProgram();
+      program.parse(argv);
```

Comment:
Is there an additional parse? I thought I moved it out of `getConfiguration()`. That way we can pass in `program` as a collaborator.